bugherd ticket #587 (#517)
SoftwareAG/c8y-docs: content/concepts/glossary.md

For details see [Device Management](/users-guide/device-management) in the *User guide*.

<a name="device-protocol"></a>
### Device protocol

> A device protocol defines how data from a specific device type should be transferred into the {{< product-c8y-iot >}} platform. It contains protocol specific information about where the data is obtained from, how to transform it, and how to store it in {{< product-c8y-iot >}}.

Reviewer: ```suggestion
A device protocol defines how data from a specific device type should be transferred into the {{< product-c8y-iot >}} platform. It contains protocol-specific information about where the data is obtained from, how to transform it, and how to store it in {{< product-c8y-iot >}}.
```

The definition of the device protocol differs based on the underlying protocol used by the device (for example, Modbus, LoRa, OPC UA or LWM2M).

> 

Reviewer: ```suggestion

For details see the [Protocol integration guide](/protocl-integration/overview).
```

For details see [Device Management](/users-guide/device-management) in the *User guide*.

<a name="device-protocol"></a>
### Device protocol
A device protocol defines how data from a specific device type should be transferred into the {{< product-c8y-iot >}} platform. It contains protocol-specific information about where the data is obtained from, how to transform it, and how to store it in {{< product-c8y-iot >}}.
The definition of the device protocol differs based on the underlying protocol used by the device (for example, Modbus, LoRa, OPC UA or LWM2M).

> For details see the [Protocol integration guide](/protocl-integration/overview).

Reviewer: ```suggestion
For details see the [Protocol integration guide](/protocol-integration/overview).

```

> For these devices the device agent will take care of the integration of the devices' data with the {{< product-c8y-iot >}} platform.

Reviewer: ```suggestion
```

For details see [Device Management](/users-guide/device-management) in the *User guide*.

<a name="device-protocol"></a>
### Device protocol

> A device protocol describes how to integrate a specific device type into the {{< product-c8y-iot >}} platform, that is, it describes how to map the device data into the {{< product-c8y-iot >}} data model.

Reviewer 2: I'd change this sentence because it is confusingly worded, partially incorrect, and misses an important point.
```suggestion
A device protocol defines how data from a specific device type should be transferred into the {{< product-c8y-iot >}} platform. It contains protocol specific information about where the data is obtained from, how to transform it, and how to store it in {{< product-c8y-iot >}}.
```

The definition of the device protocol differs based on the underlying protocol used by the device (for example, Modbus, LoRa, OPC UA or LWM2M).

> You don't need to create device protocols for devices that are natively connected to the {{< product-c8y-iot >}} platform (for example, using device SDKs, thin-edge.io or REST/MQTT interfaces).

Reviewer: ```suggestion
For devices that are natively connected to the {{< product-c8y-iot >}} platform (for example, using device SDKs, thin-edge.io or REST/MQTT interfaces), you don't need to create device protocols.
```

> For devices that are natively connected to the {{< product-c8y-iot >}} platform (for example, using device SDKs, thin-edge.io or REST/MQTT interfaces), you don't need to create device protocols.

Reviewer 2: This sentence doesn't really make sense to me. Device protocols are entirely independent from how the parent device is integrated into c8y. Can we just remove this sentence?

Reviewer: I was also wondering about the last sentence. Will remove it.

Reviewer: ```suggestion
```

For details see [Device Management](/users-guide/device-management) in the *User guide*.

<a name="device-protocol"></a>
### Device protocol
A device protocol describes how to integrate a specific device type into the {{< product-c8y-iot >}} platform, that is, it describes how to map the device data into the {{< product-c8y-iot >}} data model.

> It describes how to map the device data into the {{< product-c8y-iot >}} data model.

Reviewer: ```suggestion
```
